src/sugar3/graphics/toolbutton.py

```diff
@@ -28,6 +27,11 @@
 from sugar3.graphics.icon import Icon
 from sugar3.graphics.palette import Palette, ToolInvoker
 
+'''
+Toolbutton is a simple button which can contain only
+one event, for example, buttons in toolbars.
```

Comment:
look in, for example, https://github.com/sugarlabs/sugar/blob/master/src/jarabe/journal/journaltoolbox.py